ontology/yaml/resources/HVAC/entity_types/WSHP.yaml

```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
   - HWDT
   - POTHWSC
 
+WSHP_FDPM_CWDT_SFVSC_CDCPC_CAM_ZTC_RTM_CWDT_CDWISOVM_STM:
+  guid: "e1b4a911-00a8-42da-abaf-8eff6987ad58"
+  description: "Cooling only water source heat pump using condensor water to carry heat away from the zone.  One filter, condensor water pump control, one compressor, condensate alarm"
+  is_canonical: true
+  implements:
+  - WSHP
+  - FDPM
+  - CWDT
+  - SFVSC
+  - CDCPC
+  - CAM
+  - ZTC
```

Comment:
How does this unit control to zone air temperature? I would expect it to control to a supply or return water temperature setpoint.

Reply:
It controls to zone air, regulating the compressor to attain zone temp setpoint.  The compressor uses refrigerant to cool a condenser water loop within the unit.  The cooled condenser water flows through an air/condenser water coil.